fluid/adversarial/mnist_tutorial_fgsm.py

```diff
@@ -76,10 +76,11 @@ def main():
     att = GradientSignAttack(m)
     for data in train_reader():
         # fgsm attack
-        adv_img = att(data)
-        plt.imshow(n[0][0], cmap='Greys_r')
-        plt.show()
-        #np.save('adv_img', adv_img)
+        adversary = att(Adversary(data))
```

Comment:
The type of data is [(np.numpy, int)]，data[0][0] represents the image and data[0][1] represent the label, so it should be like this?
```python
adversary = att(Adversary(data[0][0], data[0][1]))
```

Reply:
Yep!